app/models/episode.rb

```diff
@@ -312,6 +312,7 @@ def sanitize_text
     self.subtitle = sanitize_text_only(subtitle) if subtitle_changed?
     self.summary = sanitize_links_only(summary) if summary_changed?
     self.title = sanitize_text_only(title) if title_changed?
+    self.original_guid = original_guid.strip if !original_guid.blank? && original_guid_changed?
```

Comment:
We're a bit inconsistent in this app about using setters vs validation-hooks to mutate data in this app.  But lumping this in with "sanitize stuff" makes sense to me.  Good sense!

Reply:
yeah, this didn't seem like a validation - we also strip these on import, which made me feel confident this should just get cleaned